sdk/identity/azure-identity/src/main/java/com/azure/identity/DefaultAzureCredentialBuilder.java

```diff
@@ -240,8 +240,8 @@ public DefaultAzureCredentialBuilder credentialProcessTimeout(Duration credentia
      *
      * @return An updated instance of this builder with instance discovery disabled.
      */
-    public DefaultAzureCredentialBuilder disableAuthorityValidationAndInstanceDiscovery() {
-        this.identityClientOptions.disableAuthorityValidationAndInstanceDiscovery();
+    public DefaultAzureCredentialBuilder disableInstanceDiscovery() {
```

Comment:
It would also be useful to log a warning in the `build()` method to note that this was disabled and the ramifications of disabling this. When troubleshooting, we can get the logs to see if disabling this was the reason for some issue.

Reply:
added verbose logs in client setup code.